ccf_transaction_processor/transaction_processor/pdo_tp.cpp

```diff
@@ -49,10 +49,48 @@ namespace ccfapp
             return make_success(true);
         };
 
+        //======================================================================================================
+        // gen_signing_key implementation
+        auto gen_signing_key = [this](Store::Tx& tx, const nlohmann::json& params) {
```

Comment:
Thank you for the question, unfortunately, AFAIK, there is no way for rpc to issue a local comit and wait until global commit to return the rpc. only way is to check status again to see if commit is successful or not.

Reply:
I suppose we could implement the wait loop ourselves, but it seems like that just amounts to doing `get_ledger_verifying_key()` calls anyway... Might be something to consider doing in the future, but since the responses clearly indicate how to poll the ledger by checking the status, I'm fine leaving this as-is for now.